assets/src/styles/blocks/Spreadsheet.scss

```diff
@@ -56,6 +56,12 @@ table.spreadsheet-table {
       transition: opacity .3s, transform .4s;
     }
 
+    .spreadsheet-sort-button {
```

Comment:
We also have an existing SCSS selector for the heading cells, to which you could add this as a descendant selector, without needing a class. https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/blob/36aae1ed401a88abe5570b54a9c4e29094125846/assets/src/styles/blocks/Spreadsheet.scss#L28-L37